docs/navigation-advanced.md

```diff
@@ -340,6 +341,18 @@ Tab-style navigation in Luigi can be displayed directly above the micro frontend
 - **type**: boolean
 - **description**: renders the children of the node as a horizontal navigation bar. Sub-children are not supported. When you categorize nodes you will get a drop-down menu in the horizontal navigation.
 
+## Global search
+
+The global search element can be used to trigger a search. In your configuration you can implement an own search provider and some action events like onEnter, onEscape, onInput and onSearchResultItemSelected.
+By default the search result will be rendered in a popup.
+
+Example:
```

Comment:
It would be good to add a comprehensive example here and also maybe a screenshot above.